### DISCUSSION TITLE
[Bug]: Storage usage stat is wrong.

### DISCUSSION THREAD

**Discussion by distante** [blakeblackshear/frigate#10783] [General]
I am using an shared folder from my NAS as media file for Frigate (no HA). When I see the Disk usage in Frigate UI I see a really big value (for just 3 days with recording with one camera, just on motion, in an indoor place with little motion):

<img width="508" alt="image" src="https://github.com/blakeblackshear/frigate/assets/5490201/750622a2-4d6f-4e5f-86cf-407d414dab96">

First I though I f*ck up something, but when inspect my NAS I see that the real usage is:

<img width="456" alt="image" src="https://github.com/blakeblackshear/frigate/assets/5490201/1156f051-1b49-4572-bd95-0dcb358e9367">

So, I am assuming it is a bug.

The stats endpoint is returning:

```
"storage": {
            "/dev/shm": {
                "free": 136.5,
                "mount_type": "tmpfs",
                "total": 150.0,
                "used": 13.5
            },
            "/media/frigate/clips": {
                "free": 1167893.9,
                "mount_type": "cifs",
                "total": 1944221.7,
                "used": 776327.8
            },
            "/media/frigate/recordings": {
                "free": 1167893.9,
                "mount_type": "cifs",
                "total": 1944221.7,
                "used": 776327.8
            },
            "/tmp/cache": {
                "free": 952.7,
                "mount_type": "tmpfs",
                "total": 953.7,
                "used": 1.0
            }
        },
```

I am on  version `0.13.2-6476f8a`

**Comment by NickM-27** [comment]
Like it explains in the tooltip, the storage stats show the storage usage on the entire drive not just the space used by frigate itself

**Reply by distante** [reply]
OK. Then it is a feature request? 😁

I mean, when using just the same drive where frigate is it makes sense. But when storing in external/remote storage, knowing the total usage is not that important, as the remaining and used/created by Frigate. Do not?

**Reply by NickM-27** [reply]
This was a deliberate decision, checking the specific storage usage requires spinning up drives which means HDDs would never spin down for users which is very undesirable behavior. 

Now that users have been running 0.12 / 0.13 we could potentially use the database to know how much storage is being used just for recordings but there would be no way to know about snapshots

**Reply by distante** [reply]
I see, maybe just a "calculate" button on the system view?

Maybe I am over thinking it.

**Reply by NickM-27** [reply]
the UI is currently being redone so this info might be shown differently anyway
